### DISCUSSION TITLE
Any Version Compatible.

### DISCUSSION THREAD

**Discussion by FastNinjax** [diamond-rip/Eden#231] [Ideas]
Can u make it so ur plugin can be compatible with any version more specifically 1.19+ becuz ur plugin is the best in the market currently even though it is in the beta stage. pls make it so we can use it in other versions cuz it is too good to pass.

**Comment by GoodestEnglish** [comment]
This won't be considered because compare from 1.8 to 1.19, there's a lot of changes to the bukkit api. Plus Eden used a lot of NMS (NMS means 1.8 exclusive).

1.19 also contains a new chunk system, which makes Eden arena system incompatible.

**Reply by FastNinjax** [reply]
i see but is there a chance it will become compatible with the latest versions in the future ?

**Reply by GoodestEnglish** [reply]
Prob no.
